### PR TITLE
bz1900784

### PR DESCRIPTION
**Pull request by nalhadef**
Replaces [PR38885](https://github.com/openshift/openshift-docs/pull/38885/files)

For version 4.6 only (upgrading from 4.5)

Bug: [BZ1900784](https://bugzilla.redhat.com/show_bug.cgi?id=1900784)

Direct link: https://deploy-preview-42141--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere-installer-provisioned.html#installation-vsphere-installer-infra-requirements_installing-vsphere-installer-provisioned

SME: @oarribas
QE: @jinyunma

### CODE REVIEW

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 1]
```diff
@@ -244,6 +244,12 @@ Available resources vary between clusters. The number of possible clusters withi
 == Networking requirements
 
 You must use DHCP for the network and ensure that the DHCP server is configured to provide persistent IP addresses to the cluster machines.
+
+[NOTE]
+===
+Persistent IP addresses are not available before the installation begins. Allocate a DHCP range and, after installation, manually replace the allocation with the persistent IP addresses. 
+===
```
Comment (jinyunma) [inline modules/installation-vsphere-installer-infra-requirements.adoc:251]: seems that characters "===" are also shown in doc, suggest to remove them.

**Comment by jinyunma** [comment]
LGTM

File: modules/installation-vsphere-installer-infra-requirements.adoc [hunk 2]
```diff
@@ -257,10 +263,11 @@ It is recommended that each {product-title} node in the cluster must have access
 [discrete]
 [id="installation-vsphere-installer-infra-requirements-_{context}"]
 === Required IP Addresses
-An installer-provisioned vSphere installation requires two static IP addresses:
+An installer-provisioned vSphere installation requires these static IP addresses:
 
 * The **API** address is used to access the cluster API.
 * The **Ingress** address is used for cluster ingress traffic.
+* The control plane node addresses are used when upgrading a cluster from version 4.5 to 4.6.
```
Comment (jeana-redhat) [inline modules/installation-vsphere-installer-infra-requirements.adoc:270]: Unclear here why "API" and "Ingress" are bold - are they field labels in a GUI or something? If so, is "control plane node" also a label? It is possible that the bold should not be consistent on these three items, but want to verify that makes sense.

**Review by jeana-redhat — COMMENTED** [review]
Just one question about formatting - let me know if that's as it should be or if you need to change it and I will merge or not accordingly :)

File: modules/installation-vsphere-installer-infra-requirements.adoc (hunk 2 above)
Comment (nalhadef) [reply modules/installation-vsphere-installer-infra-requirements.adoc:270]: Thanks, @jeana-redhat . I removed the bold from "API" and "Ingress."

**Pull request merged**